@uportal/dashboard-carousel/src/components/DashboardCarousel.vue

```diff
@@ -85,6 +84,13 @@ export default {
       type: String,
       default: '/uPortal/api/v4-3/dlm/layout.json',
     },
+    slickOptions: {
+      type: String | Object,
```

Comment:
```
88:13  error    The "slickOptions" property should be a constructor  vue/require-prop-type-constructor
```

https://travis-ci.org/uPortal-contrib/uPortal-web-components/jobs/542909832#L499

Reply:
```suggestion
      type: [String , Object],
```

may work better
https://vuejs.org/v2/guide/components-props.html#Prop-Validation